### ISSUE TITLE
Broken compilation with GHC 7.6.3 because EdisonCore doesn't support it

### ISSUE THREAD

**Issue by asr** [agda/agda#1803]
```
$ make install-bin
...
Preprocessing library EdisonCore-1.3.1...
...
src/Data/Edison/Coll/EnumSet.hs:182:14:
    Not in scope: `finiteBitSize'
Failed to install EdisonCore-1.3.1
```

I filed an issue for EdisonCore (see https://github.com/robdockins/edison/issues/4).

**Comment by asr** [comment]
@nad Please take ownership of this issue.

**Comment by andreasabel** [comment]
Why did https://travis-ci.org/agda/agda/builds/105359554 succeed even before this fix?
On a quick glance, travis did not attempt to recompile agda with ghc-7.6, but used some already compiled version!?

**Comment by asr** [comment]
Because

i) before the the fix the dependence was `EdisonCore == 1.3.1.*` and

ii) EdisonCore 1.3.1.1 fixing the issue was released

So Travis chose the right version of EdisionCore.

**Comment by asr** [comment]
> On a quick glance, travis did not attempt to recompile agda with ghc-7.6, but used some already compiled version!?

No. Travis compiles a fresh version Agda (expand lines 413, 636, etc., in the [105359554](https://travis-ci.org/agda/agda/builds/105359554) Travis build.
